community/recruitment.md

```diff
@@ -0,0 +1,20 @@
++++
+toc=true
++++
+
+# Recruitment
+
+## Whom do we recruit?
+We recruit UG 1st years in the team, preferably after they have spent one semester on the campus. We can also recruit any 2nd year in their third semester, based on recommendations by the members.
+
+## How do we recruit?
+For more detailed description, refer [Freshers' Selection](/docs/events/freshers-selection)
+
+## Do we accept M.Tech and Ph.D students?
+By acceptance, we mean that they will hold a number of responsibilities
```

Comment:
```suggestion
No. By acceptance, we mean that they will hold a number of responsibilities
```

Reply:
Great! I was also thinking that our stand is not clear :smile: